apps/server/src/modules/deletion-console/uc/batch-deletion.uc.ts

```diff
@@ -1,4 +1,6 @@
 import { Injectable } from '@nestjs/common';
+import { AccountService } from '@src/modules/account';
+import { UserService } from '@src/modules/user';
```

Comment:
keep to convention for imports from modules @modules/....

Reply:
done